### PR TITLE
Fixed consistency of top banner by switching to make_banner function.

### PR DESCRIPTION
**Pull request by Jchuchla**
Fixed consistency of top banner by switching to make_banner function.

Fixed column layout to show in four columns

Created CSS class for column class/group headers.

Fixed sizing and centering of logo shown when all cars checked in.

### CODE REVIEW

**Comment by Jchuchla** [comment]
This PR replaces the previous PR #226

File: website/kiosks/please-check-in.kiosk [hunk 1]
```diff
@@ -15,32 +15,37 @@ $(function () { setInterval(function() { location.reload(true); }, 30000); });
...
-  border-bottom: 2px solid black;
-}
 .wanted_container {
-  column-width: 250px;
   position: absolute;
-  top: 100px;
-  bottom: 0;
-}
-.wanted_container ul {
-    -moz-column-count: 5;
```
Comment (jeffpiazza) [inline website/kiosks/please-check-in.kiosk:28]: These -moz- and -webkit- attribute prefixes were added in #227, and I'd like to keep them for wider compatibility.

**Comment by Jchuchla** [comment]
I tested against edge, chrome and firefox on windows and I got consistency without the extraneous settings in it.  Don’t have a mac to test against.  I plan on spinning up a pi later to test as a kiosk.  Both of these properties have been widely supported for quite some time
"column-gap" | Can I use... Support tables for HTML5, CSS3, etc<https://caniuse.com/?search=column-gap>
"column-count" | Can I use... Support tables for HTML5, CSS3, etc<https://caniuse.com/?search=column-count>

There’s also the difference in that he put the columns on the ul element where I put it on the container instead.  I found that setting those properties on the ul element gave inconsistent results.

In general it’s better to avoid browser specific stuff where it isn’t truly necessary.  Did the other contributor mention which browser/platform he was testing against and found these necessary?

Does this project have a stated browser version support list?  I’m assuming pretty wide platform support, but not sure about how old you like to go.

**Comment by burnclouds** [comment]
Putting the properties on the container is probably best.
As for the WebKit and moz specific they are more for rendering that doesn't
support html5. So I suspect jchuchla has a completely acceptable solution.
And is better than mine.

**Pull request merged**